orion/pkg/repos/migrations/000007_create_table_sessions.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE sessions (
+	id int unsigned NOT NULL AUTO_INCREMENT,
+	created_at datetime NOT NULL,
+	updated_at datetime NOT NULL,
+	deleted_at datetime,
+	class_id varchar(192) NOT NULL,
+	starts_at datetime NOT NULL,
+	ends_at datetime NOT NULL,
+	canceled boolean NOT NULL DEFAULT 0,
+	notes text NOT NULL,
```

Comment:
ah sorry forgot to mention. notes here can actually be NULLABLE. `notes` are little blurbs that the admin can use to display to students like "Room change today to 124!"

Just make this Nullable here, but don't worry about the whole NullString debacle yet. Wait for #72 to merge, which will make it easier to use NullStrings in places.